act-rules-format.bs

```diff
@@ -321,7 +313,38 @@ Each new release of an ACT Rule MUST be identifiable with either a date or a ver
 </div>
 
 
-Issues List {#updates-issues}
+Optional items {#optional-items}
+================================
+
+An ACT Rule can contain additional information that is useful, but not critical, for the reader to understand the background and context of a rule, or any information that is relevant. This can include, but is not limited to, one or more of the following:
+
+
+Accessibility Support {#accessibility-support}
+----------------------------------------------
+Content can be designed to rely on the support for particular accessibility features by different assistive technologies and user agents. For example, content using a particular WAI-ARIA [[WAI-ARIA]] feature relies on that feature to be supported by assistive technologies and user agents. This content would not work for assistive technologies and user agents that do not support WAI-ARIA. WCAG [[WCAG]] provides a definition for [accessibility supported](https://www.w3.org/TR/WCAG20/#accessibility-supporteddef) use of a Web technology.
+ 
+An ACT Rule SHOULD include known limitations on accessibility support. For example, an atomic rule that checks for a particular accessibility feature that is known not to be widely supported by assistive technologies and web browsers, or a composite rule that includes atomic rules with known accessibility support limitations.
```

Comment:
Accessibility support falls under "should". I don't think it can be treated as optional.

Reply:
In the ACT Rules Structure section it said "Accessibility Support (optional)".
Do you disagree with that too?

Some of the others say ", if any" same place. Accessibility Support is the only one that had "(optional)" on it.